terraform/schema.yaml

```diff
@@ -458,6 +459,19 @@ variables:
                 - ${subnet_span}
                 - "Regional Subnet"
 
+  place_all_compute_in_same_ad:
```

Comment:
Can this be made visible only if the node count is more than one ? Since this option won't matter if the node count is 1.

Reply:
updated the PR

Reply:
Reverted the change